scripts/opx_build

```diff
@@ -17,7 +17,15 @@
 
 set -e
 
-DIST=jessie
+DIST=$(lsb_release --codename --short)
```

Comment:
@mabnhdev, I've tested this and using "gbp buildpackage" works with a Jessie container too.  Can you please rework this change so that's used unconditionally?